bin/config.toml

```diff
@@ -187,31 +189,116 @@ address = "0.0.0.0:8443"
 # this option is incompatible with public_address
 # expect_proxy = false
 
-# supported TLS versions. Possible values are "SSLv2", "SSLv3", "TLSv1",
-# "TLSv1.1", "TLSv1.2", "TLSv1.3". Defaults to `["TLSv1.2", "TLSv1.3"]`
+# Supported TLS versions. Possible values are "SSLv2", "SSLv3", "TLSv1", "TLSv1.1", "TLSv1.2", "TLSv1.3".
+# Defaults to `["TLSv1.2", "TLSv1.3"]`. Besides, `rustls` tls provider only support "TLSv1.2" and "TLSv1.3" values.
 tls_versions = ["TLSv1.2", "TLSv1.3"]
 
-# cipher combinations used by OpenSSL, you can retrieve cipher suites secure and
-# recomended here: https://ciphersuite.info/cs/?security=secure
-cipher_list = "DHE-RSA-AES128-CCM:ECDHE-ECDSA-CHACHA20-POLY1305:DHE-RSA-AES128-GCM-SHA256:ECDHE-PSK-CHACHA20-POLY1305:DHE-PSK-AES128-GCM-SHA256:DHE-PSK-CHACHA20-POLY1305:DHE-RSA-AES256-CCM8:DHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-AES256-CCM:DHE-RSA-AES256-CCM:ECDHE-ECDSA-AES128-GCM-SHA256:DHE-PSK-AES128-CCM8:DHE-PSK-AES256-CCM:TLS_AES_128_CCM_SHA256:ECDHE-ECDSA-AES128-CCM8:DHE-DSS-AES128-GCM-SHA256:DHE-RSA-CHACHA20-POLY1305:ECDHE-RSA-AES256-GCM-SHA384:TLS_AES_128_GCM_SHA256:DHE-DSS-AES256-GCM-SHA384:ECDHE-ECDSA-AES256-CCM8:DHE-PSK-AES256-GCM-SHA384:DHE-PSK-AES256-CCM8:DHE-RSA-AES128-CCM8:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-ECDSA-AES128-CCM:ECDHE-RSA-AES128-GCM-SHA256:DHE-PSK-AES128-CCM:TLS_AES_256_GCM_SHA384"
-
-# cipher combinations used by rustls
-# ciphersuites names, specified in https://docs.rs/rustls/latest/rustls/static.ALL_CIPHER_SUITES.html
-# defaults to all supported ciphersuites
-rustls_cipher_list = [
-  # TLS 1.3 cipher suites
-  "TLS13_AES_256_GCM_SHA384",
-  "TLS13_AES_128_GCM_SHA256",
-  "TLS13_CHACHA20_POLY1305_SHA256",
-  # TLS 1.2 cipher suites
-  "TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384",
-  "TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256",
-  "TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305_SHA256",
-  "TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384",
-  "TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256",
-  "TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305_SHA256",
+# TLS ciphers combinations can be retrieve using cipher suites website by query secure and recomended ones.
```

Comment:
*retrieved*

"ciphered suites website" => what is this? one website? 

This lacks clarity. Please rewrite this in the style of "TLS ciphers can be retrieved on cipher suite websites, here is one we recommend: https://ciphersuite.info/cs/?security=secure"